### PR TITLE
Fix: The background color of the tooltip needs to be #000000. 

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------

Uniform the background color of the tooltip to be `#000000`, and the opacity to be `0.8`.

image

Figma: https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=67%3A1378&t=DaTd93SVt1TYisnq-0

Ref: https://github.com/Magickbase/godwoken_explorer/issues/1216

Check List
------

### Test
e2e Test
### Task
none

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```
Comment (Keith-CY) [inline .gitignore:29]: Better to be placed in the `dependencies` section

**Comment by Keith-CY** [comment]
`package-lock.json` should not be updated because no dependencies have been changed

**Comment by JeffreyMa597** [comment]
> `package-lock.json` should not be updated because no dependencies have been changed.

@Keith-CY I checked these changes. Should I roll back them?
image

**Comment by Keith-CY** [comment]
> > `package-lock.json` should not be updated because no dependencies have been changed.
> 
> @Keith-CY I checked these changes. Should I roll back them? image

Discarding changes in `package-lock.json` is fine

**Review by Keith-CY — APPROVED** [review]

**Review by FrederLu — APPROVED** [review]

**Pull request merged**